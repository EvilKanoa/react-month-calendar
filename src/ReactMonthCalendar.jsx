import React from 'react';

const ReactMonthCalendar = ({ name }) => <span>{name}</span>;

export default ReactMonthCalendar;
