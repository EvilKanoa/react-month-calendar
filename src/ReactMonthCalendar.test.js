import React from 'react';
import { mount } from 'enzyme';

import ReactMonthCalendar from './ReactMonthCalendar';

describe('ReactMonthCalendar', () => {
  describe('render tests', () => {
    it('renders with default props', () => {
      const wrapper = mount(<ReactMonthCalendar />);
      expect(wrapper).toMatchSnapshot();
    });
  });
});
