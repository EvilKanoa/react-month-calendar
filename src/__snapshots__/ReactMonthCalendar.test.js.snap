// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ReactMonthCalendar render tests renders with default props 1`] = `
<ReactMonthCalendar>
  <span />
</ReactMonthCalendar>
`;
